drivers/fpga/xrt/include/subdev_id.h

```diff
@@ -15,23 +15,23 @@
  * of a specific instance of a subdev driver.
  */
 enum xrt_subdev_id {
-	XRT_SUBDEV_GRP = 0,
-	XRT_SUBDEV_VSEC = 1,
-	XRT_SUBDEV_VSEC_GOLDEN = 2,
-	XRT_SUBDEV_DEVCTL = 3,
-	XRT_SUBDEV_AXIGATE = 4,
-	XRT_SUBDEV_ICAP = 5,
-	XRT_SUBDEV_TEST = 6,
-	XRT_SUBDEV_MGMT_MAIN = 7,
-	XRT_SUBDEV_QSPI = 8,
-	XRT_SUBDEV_MAILBOX = 9,
-	XRT_SUBDEV_CMC = 10,
-	XRT_SUBDEV_CALIB = 11,
-	XRT_SUBDEV_CLKFREQ = 12,
-	XRT_SUBDEV_CLOCK = 13,
-	XRT_SUBDEV_SRSR = 14,
-	XRT_SUBDEV_UCS = 15,
-	XRT_SUBDEV_NUM = 16, /* Total number of subdevs. */
+	XRT_SUBDEV_GRP = 1,
+	XRT_SUBDEV_VSEC = 2,
+	XRT_SUBDEV_VSEC_GOLDEN = 3,
+	XRT_SUBDEV_DEVCTL = 4,
+	XRT_SUBDEV_AXIGATE = 5,
+	XRT_SUBDEV_ICAP = 6,
+	XRT_SUBDEV_TEST = 7,
+	XRT_SUBDEV_MGMT_MAIN = 8,
+	XRT_SUBDEV_QSPI = 9,
+	XRT_SUBDEV_MAILBOX = 10,
+	XRT_SUBDEV_CMC = 11,
+	XRT_SUBDEV_CALIB = 12,
+	XRT_SUBDEV_CLKFREQ = 13,
+	XRT_SUBDEV_CLOCK = 14,
+	XRT_SUBDEV_SRSR = 15,
+	XRT_SUBDEV_UCS = 16,
+	XRT_SUBDEV_NUM = 17, /* Total number of subdevs. */
```

Comment:
why do you want to change the value of these enums? XRT_SUBDEV_NUM is the total num of subdevs and should still be 16?